share/p2p/peers/manager.go

```diff
@@ -322,37 +342,54 @@ func (m *Manager) hashIsBlacklisted(hash share.DataHash) bool {
 	return m.blacklistedHashes[hash.String()]
 }
 
+func (m *Manager) validatedPool(hashStr string) *syncPool {
```

Comment:
can we call this `validatePool` ?

Reply:
The main purpose is to get validated pool here and a side affect is making it validated if not